app/test/std_sdk_test.go

```diff
@@ -153,61 +149,12 @@ func (s *StandardSDKIntegrationTestSuite) TestStandardSDK() {
 				valopAccAddr := testfactory.GetAddress(s.cctx.Keyring, account)
 				valopAddr := sdk.ValAddress(valopAccAddr)
 				msg, err := stakingtypes.NewMsgCreateValidator(
-					valopAddr,
+					valopAddr.String(),
 					pv.PrivKey.PubKey(),
-					sdk.NewCoin(app.BondDenom, sdk.NewInt(1)),
+					sdk.NewCoin(app.BondDenom, math.NewInt(1)),
 					stakingtypes.NewDescription("taco tuesday", "my keybase", "www.celestia.org", "ping @celestiaorg on twitter", "fake validator"),
-					stakingtypes.NewCommissionRates(sdk.NewDecWithPrec(6, 0o2), sdk.NewDecWithPrec(12, 0o2), sdk.NewDecWithPrec(1, 0o2)),
-					sdk.NewInt(1),
-				)
-				require.NoError(t, err)
-				return []sdk.Msg{msg}, account
-			},
-			expectedCode: abci.CodeTypeOK,
-		},
-		{
-			name: "create continuous vesting account with a start time in the future",
-			msgFunc: func() (msgs []sdk.Msg, signer string) {
-				vestAccName := "vesting"
-				_, _, err := s.cctx.Keyring.NewMnemonic(vestAccName, keyring.English, "", "", hd.Secp256k1)
-				require.NoError(t, err)
-				sendAcc := s.unusedAccount()
-				sendingAccAddr := testfactory.GetAddress(s.cctx.Keyring, sendAcc)
-				vestAccAddr := testfactory.GetAddress(s.cctx.Keyring, vestAccName)
-				msg := vestingtypes.NewMsgCreateVestingAccount(
-					sendingAccAddr,
-					vestAccAddr,
-					sdk.NewCoins(sdk.NewCoin(app.BondDenom, sdk.NewInt(1000000))),
-					time.Now().Add(time.Hour).Unix(),
-					time.Now().Add(time.Hour*2).Unix(),
-					false,
-				)
-				return []sdk.Msg{msg}, sendAcc
-			},
-			expectedCode: abci.CodeTypeOK,
-		},
```

Comment:
same question as above

Reply:
If we bring back the vesting module we would be able to bring this back yes.